modules/transformer-batch/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/batch/badrows/WiderowFileSink.scala

```diff
@@ -45,13 +44,9 @@ final class WiderowFileSink(
   private def openFile(
     partitionIndex: Int
   ): FSDataOutputStream = {
-    val directory = Paths.get(
-      outputPath.toString,
-      folderName,
-      "output=bad"
-    )
+    val directory = s"$outputPath/$folderName/output=bad"
```

Comment:
I have no problems with this if it works.  But did you consider using [this existing code](https://github.com/snowplow/snowplow-rdb-loader/blob/5.4.0/modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/cloud/BlobStorage.scala#L100) for concatenating the directory names?